Activities/Activities.Python.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30621.155
```

Comment:
this change (and a few other) doesn't seem related to the jira tickets that are linked to this ticket

Reply:
source files from the shared projects were used but not included in the solution